pkg/util/secretcode/oob_otp.go

```diff
@@ -17,7 +18,8 @@ func (OOBOTPSecretCodeType) Generate() string {
 }
 
 func (OOBOTPSecretCodeType) Compare(a, b string) bool {
-	return subtle.ConstantTimeCompare([]byte(a), []byte(b)) == 1
+	formatedCode := strings.TrimSpace(a)
```

Comment:
This method treats its argument commutatively. Trim both `a` and `b`.

`s/formatedCode/formattedCode`